packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -246,9 +262,22 @@ const BaseChart = ({
             />
           </HotKeys>
         </Resizable>
-        <div className='base-chart-timestamp' style={timestampStyle}>
-          <span>{timestampStart}</span>
-          <span>{timestampEnd}</span>
+        <div className='base-chart-timestamp-container' style={timestampStyle}>
```

Comment:
could you move this whole timestamp container section into its own file

Reply:
updated